src/Networking/NexusMods.Networking.ModUpdates/Structures/ModId.cs

```diff
@@ -0,0 +1,10 @@
+using TransparentValueObjects;
+namespace NexusMods.Networking.ModUpdates.Structures;
+
+/// <summary>
+/// Represents the unique ID of an individual mod in the Nexus Backend.
+/// This <see cref="ModId"/> is specific to the <see cref="GameId"/> it belongs to;
+/// forming a composite key. 
+/// </summary>
+[ValueObject<uint>] // Do not modify. Unsafe code relies on this.
+public readonly partial struct ModId { }
```

Comment:
why not use the existing ModId?

Reply:
I did that in the following PR. This was noted in the opening post.

I didn't do it here because some of the changes required would touch external code, meaning a lot of files would get changed in small ways.